### PR TITLE
feat: add gloadss opcode

### PR DESCRIPTION
**Pull request by vuvth**
## Proposed Changes

+ Add gloadss(tealv6) opcode.
+ Unit tests.

## TODO
- [x] Update changelog after PR approval.

### CODE REVIEW

File: packages/runtime/src/parser/parser.ts [hunk 1]
```diff
@@ -243,7 +243,8 @@ opCodeMap[5] = {
 };
 
 opCodeMap[6] = {
-  ...opCodeMap[5]
+  ...opCodeMap[5],
+  gloadss: Gloadss
 };
 
 // list of opcodes that require one extra parameter than others: `interpreter`.
```
Comment (robert-zaremba) [inline packages/runtime/src/parser/parser.ts:250]:
```suggestion
// list of opcodes with exactly one parameter.
```

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -2712,6 +2712,29 @@ export class Gloads extends Gload {
...
+export class Gloadss extends Gload {
+  /**
+   * Stores scratch space index number according to argument passed.
+   * @param args Expected arguments: [index number]
+   * @param line line number in TEAL file
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
+    // "11" is mock value, will be updated when poping from stack in execute
+    super(["11", "11", ...args], line, interpreter);
```
Comment (robert-zaremba) [inline packages/runtime/src/interpreter/opcode-list.ts:2727]: Let's improve the docs and style based on https://github.com/scale-it/algo-builder/pull/608

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 3]
```diff
@@ -2712,6 +2712,29 @@ export class Gloads extends Gload {
...
+  }
+
+  execute (stack: TEALStack): void {
+    this.assertMinStackLen(stack, 2, this.line);
+    this.scratchIndex = Number(this.assertBigInt(stack.pop(), this.line));
+    this.txIndex = Number(this.assertBigInt(stack.pop(), this.line));
+    super.execute(stack);
+  }
```
Comment (robert-zaremba) [inline packages/runtime/src/interpreter/opcode-list.ts:2735]: nice trick :+1:

**Review by robert-zaremba — APPROVED** [review]
let's rebase it after https://github.com/scale-it/algo-builder/pull/608 will be merged

File: packages/runtime/src/interpreter/opcode-list.ts (hunk 2 above)
Comment (vuvth) [reply packages/runtime/src/interpreter/opcode-list.ts:2732]: > Let's improve the docs and style based on #608

I update code base @robert-zaremba

Comment (robert-zaremba) [reply packages/runtime/src/interpreter/opcode-list.ts:2732]: great! let's fix the test and merge it!

**Pull request merged**